CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Remove `TxGroup.assignGroupID(Transaction[] txns, Address address)` in favor
   of `TxGroup.assignGroupID(Address address, Transaction ...txns)`.
 * Remove `Account.transactionWithSuggestedFeePerByte` in favor of `Account.setFeeByFeePerByte`.
+* Remove deprecated methods in `Transaction.java`, mark public `Transaction` constructor as hidden in favor of `builder.transaction`.
```

Comment:
@ahangsu Optional - Where possible, I prefer a fully qualified package name to minimize confusion.  Though I don't feel strongly.

```suggestion
* Remove deprecated methods in `Transaction.java`, mark public `Transaction` constructor as hidden in favor of `com.algorand.algosdk.builder.transaction`.
```